Description: Fix GPL license to be project consistent
 Update QuickNote source file to be GPL-3+
Author: David Mohammed <fossfreedom@ubuntu.com>
Last-Update: 2008-09-22
Bug: https://github.com/UbuntuBudgie/budgie-extras/issues/94
Applied-Upstream: commit: acb38595e4c1c08a673d1fe61e5bde9c272946f3
---
 budgie-quicknote/QuickNoteApplet.vala | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/budgie-quicknote/QuickNoteApplet.vala b/budgie-quicknote/QuickNoteApplet.vala
index 476fb2a..07098ed 100644
--- a/budgie-quicknote/QuickNoteApplet.vala
+++ b/budgie-quicknote/QuickNoteApplet.vala
@@ -1,9 +1,10 @@
 using Gtk;
 /*
- * Copyright © 2017-2018 Ubuntu Budgie Developers
+ * Copyright © 2018 Ubuntu Budgie Developers
+ *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
+ * the Free Software Foundation; either version 3 of the License, or
  * (at your option) any later version.
 */

--
2.17.1

